sdcm/tester.py

```diff
@@ -233,16 +233,23 @@ def setUp(self):
         self.params['update_db_packages'] = download_dir_from_cloud(update_db_packages)
 
         self.init_resources()
-        self.set_seeds()
 
-        # Init seed nodes
-        self.db_cluster.wait_for_init(node_list=self.db_cluster.seed_nodes)
+        def init_nodes(clus):
```

Comment:
`clus` -> `_cluster`

Reply:
I called it first "cluster" and commit failed with :
```
hookid: pylint

************* Module sdcm.tester
W:237,23: Redefining name 'cluster' from outer scope (line 43) (redefined-outer-name)

```